### PR TITLE
Theta Wave N24

### PR DESCRIPTION
**Pull request by cdsupina**
#719

### CODE REVIEW

File: content/news/024/index.md [hunk 1]
```diff
@@ -278,6 +278,28 @@ Changes and improvements from the last month:
...
 [fish-fight]: https://twitter.com/fishfightgame
 
+### [Theta Wave]
+
+[![Amethyst to Bevy](theta-wave.gif)][Bevy Theta Wave]
+_Left: Theta Wave with Amethyst, Right: Theta Wave with Bevy (so far)_
+
+[Theta Wave] is an open-source space shooter game by developers [@micah_tigley] and
+[@carlosupina]. It is one of the showcase games for the [Amethyst Engine]. In
+the past month, we finished the ["Organization"] update and
```
Comment (17cupsofcoffee) [inline content/news/024/index.md:288]:
```suggestion
the past month, they finished the ["Organization"] update and
```

File: content/news/024/index.md [hunk 2]
```diff
@@ -278,6 +278,28 @@ Changes and improvements from the last month:
...
+made the decision to start working on porting the game to the Bevy engine. You can
+find the Bevy version of Theta wave [here][Bevy Theta Wave].
+
+Progress on this port is going strong you find the issue for the port
+[here][Bevy Port Issue].
```
Comment (17cupsofcoffee) [inline content/news/024/index.md:293]:
```suggestion
Progress on this port is going strong - you can find the GitHub issue for
the port [here][Bevy Port Issue].
```

File: content/news/024/index.md [hunk 3]
```diff
@@ -278,6 +278,28 @@ Changes and improvements from the last month:
...
+### [Theta Wave]
+
+[![Amethyst to Bevy](theta-wave.gif)][Bevy Theta Wave]
+_Left: Theta Wave with Amethyst, Right: Theta Wave with Bevy (so far)_
+
+[Theta Wave] is an open-source space shooter game by developers [@micah_tigley] and
+[@carlosupina]. It is one of the showcase games for the [Amethyst Engine]. In
+the past month, we finished the ["Organization"] update and
+made the decision to start working on porting the game to the Bevy engine. You can
+find the Bevy version of Theta wave [here][Bevy Theta Wave].
```
Comment (17cupsofcoffee) [inline content/news/024/index.md:290]:
```suggestion
find the Bevy version of Theta Wave [here][Bevy Theta Wave].
```

**Review by 17cupsofcoffee — COMMENTED** [review]
Content looks good, just making a few wording tweaks.

The GIF is a bit big, though - could you replace it with this optimized version? I generated it with the 'optimize transparency' option on https://ezgif.com/optimize - this works really well for your game, as a lot of the image is static from frame to frame.

theta-wave

**Comment by cdsupina** [comment]
I didn't realize you put it through the optimizer already. I put it through on my own and put that gif in its place.

**Review by 17cupsofcoffee — APPROVED** [review]
That works too :)

**Pull request merged**